Library/PKPaymentRequest+Helpers.swift

```diff
@@ -9,35 +9,34 @@ extension PKPaymentRequest {
     allRewardsTotal: Double,
     additionalPledgeAmount: Double,
     allRewardsShippingTotal: Double,
-    merchantIdentifier: String,
-    env: Environment = AppEnvironment.current
```

Comment:
This was always the default value, so just making it explicit. I'm also checking a feature flag in here which implicitly uses `AppEnvironment.current`.